victron_ble/exceptions.py

```diff
@@ -0,0 +1,6 @@
+class UnknownDevice(Exception):
```

Comment:
Can you suffix these new exception types with Error to match the PEP8 convention?

Reply:
Good catch! done